src/chrome/content/rules/hsbc.com.hk.xml

```diff
@@ -8,9 +8,13 @@
 		- assetmanagement.hsbc.com.hk
 		- business.hsbc.com.hk
 
+		SSL error:
+		- www.security.online-banking.hsbc.com.hk
```

Comment:
This might be specific to Travis, I'm not able to reproduce it. Would you like to try again and whitelist it?

Reply:
Look like the problem is fixed on the server side after the build. thanks.